service/pom.xml

```diff
@@ -232,9 +234,77 @@
     </repositories>
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.12.1</version>
+                <executions>
+                    <execution>
+                        <id>install node and npm</id>
+                        <goals>
+                            <goal>install-node-and-npm</goal>
+                        </goals>
+                        <configuration>
```

Comment:
please add the following so that we can download from an alternate location, and specify our npm version if need be.

It would also be good to add information about these configuration options in a top level README.

```
                            <nodeDownloadRoot>${node.downloadRoot}</nodeDownloadRoot>
                            <npmDownloadRoot>${npm.downloadRoot}</npmDownloadRoot>
                            <npmVersion>${npm.version}</npmVersion>
```